packages/sdk/src/lib/dataProtectorSharing/getResultFromCompletedTask.ts

```diff
@@ -2,72 +2,97 @@ import { decryptResult } from 'iexec/utils';
 import JSZip from 'jszip';
 import { getSavedKeyPair } from '../../utils/indexedDb.js';
 import { privateAsPem } from '../../utils/rsa.js';
-import { throwIfMissing } from '../../utils/validators.js';
+import {
+  pathSchema,
+  taskIdSchema,
+  throwIfMissing,
+  validateOnStatusUpdateCallback,
+} from '../../utils/validators.js';
 import {
   GetResultFromCompletedTaskParams,
   GetResultFromCompletedTaskResponse,
+  GetResultFromCompletedTaskStatuses,
+  OnStatusUpdateFn,
 } from '../types/index.js';
 import { IExecConsumer } from '../types/internalTypes.js';
 
 export const getResultFromCompletedTask = async ({
   iexec = throwIfMissing(),
+  path,
```

Comment:
`taskId` seems more "important" to me, I'd put it before this new `path` arg.

Reply:
yeah i agree, moreover taskId is the only one that is not optional
Commit: e8a2427a9c2cba003785ce31f939a7daa9f719a2